### ISSUE TITLE
log_loss is calculated on class predictions, not probabilities

### ISSUE THREAD

**Issue by arsenyinfo** [rasbt/mlxtend#176]
According to source code, currently `mlxtend.plotting.plot_learning_curves` always calls estimator's `predict` method which is not good approach for metrics like log loss.

It may be useful to use more high level abstraction like `sklearn.metrics.scorer.make_scorer` instead of metric itself.

P.S. It looks like docstring is a bit inaccurate: it contains "Plots learning curves of a classifier", but the function is suitable for regressors too, isn't it?

**Comment by rasbt** [comment]
Thanks for pointing that out. I agree, using `sklearn.metrics.scorer.make_scorer` would definitely be the better way to go about this. Will have a look at that

**Comment by soumyadsanyal** [comment]
Is anyone working on this? I'd be happy to give this a go.

**Comment by rasbt** [comment]
Not yet :). I wanted to go back to things now that the busy semester is over, but if you'd like to work on it that would be nice :).

**Comment by soumyadsanyal** [comment]
@rasbt addressed in PR #190
